aait/backend/group-2A/Application/Profiles/MappingProfile.cs

```diff
@@ -9,7 +11,18 @@ public class MappingProfile : AutoMapper.Profile
 {
 
     public MappingProfile(){
+        
         CreateMap<Post, PostDto>().ReverseMap();
+        CreateMap<Post, UpdatePostDto>().ReverseMap();
+        CreateMap<Post, CreateUserDTO>().ReverseMap();
+        
+        CreateMap<User, UserDto>();
+        CreateMap<User, CreateUserDTO>();
```

Comment:
CreateMap<User, CreateUserDTO>() and CreateMap<User, UpdatePostDto>() should be changed to CreateMap<CreateUserDTO,User>() and CreateMap<UpdatePostDto,User>() mappings